### DISCUSSION TITLE
Spellpoints & Resources

### DISCUSSION THREAD

**Discussion by Liviu50** [misthero/dnd5e-spellpoints#77] [Q&A]
From my knowledge resource mod was fixed for the v3. 
Can the module be re-enabled to use the resource in the char sheet instead of the "item" created by the module?
It would be nice to have the option at least :) cause i was using my resources before the newest update and now the magic points are no longer deducted.
Is there any chance to add the option to choose if to use resource or item or similar please?

**Comment by misthero** [comment]
what resource mod? 
You can keep using the old version until resources get eliminated for good, just right click and lock it to prevent updating.
![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/170a74e0-0328-427f-8d23-5b923f436828)

the latest version working with resources is here: https://github.com/misthero/dnd5e-spellpoints/releases/tag/v1.5.3

**Comment by misthero** [comment]
also if magic points are not deducted for you that could be a bug, can you help me replicate your issue to investigate ?

**Comment by Liviu50** [comment]
Well i simply have the mod 5e sheet resources plus, which allows me to add up to 20 resources to the char sheet.
That module was adapted to the new version from my knowledge and i am using that one to define MP or Spellpoints or SP whatever into one of the resources...of course it does not take it out when i cast a spell tho.

**Comment by misthero** [comment]
> Well i simply have the mod 5e sheet resources plus, which allows me to add up to 20 resources to the char sheet. That module was adapted to the new version from my knowledge and i am using that one to define MP or Spellpoints or SP whatever into one of the resources...of course it does not take it out when i cast a spell tho.

like I said, if you want to keep using resources you should downgrade the module to the version I linked and lock it to avoid unwanted update.

just remember that resources have been "deprecated" and sooner or later they will be gone.

**Comment by misthero** [comment]
I'm moving this to the discussions so it will stay for future reference
